.github/workflows/bump_dependencies.yml

```diff
@@ -9,12 +9,23 @@ jobs:
     if: startsWith(github.repository, 'spinnaker/')
     runs-on: ubuntu-latest
     steps:
+    # Halyard releases are separate from Spinnaker releases so we need to
+    # exclude Halyard from bump deps when targeting services release-* branches.
+    - name: decide bumpdep target repositories
+      id: bumpdep_targets
+      run: |
+        REPOS=clouddriver,echo,front50,gate,igor,keel,orca
+        if [ "${{ github.event.client_payload.branch }}" == "  origin/master"]; then
```

Comment:
The whitespace here rubs me the wrong way, but that's for another day.

Reply:
Cheers. Me too. For some reason that's how it comes through. We could regex match instead but I went with simpler option.